fobi_custom/plugins/form_elements/fields/types.py

```diff
@@ -48,14 +43,39 @@
 
 # A list of types with their corresponding ACS variables.
 TYPES_TO_ACS_VARIABLES = {
-    'age_observational': 'age_and_sex',
-    'gender_observational': 'age_and_sex',
-    'age_intercept': 'age_and_sex',
-    'gender_intercept': 'age_and_sex',
-    'income': 'household_income',
-    'education': 'educational_attainment',
-    'race': 'race',
-    'employment': 'employment_status',
-    'household_tenure': 'household_tenure',
-    'transportation': 'means_of_transit'
```

Comment:
The prewritten question here is "How did you travel here?," which I'm guessing is not the same phrasing as the census question. What do you think about including a note with charts that include Census data to say where the information is coming from, including the framing of the question?

Reply:
Good point. We had also talked about adding a similar caveat for racial data, since ACS doesn't record Hispanic/Latino as a race.

Right now, the pull request assumes that it's the responsibility of the user to note these kinds of caveats in the "short description" field in the DAD. Adding custom notes seems like it would have the advantage of taking that step off of the user's plate, but at the cost that the notes would either have to 1) be fixed by us or 2) exposed for list/detail/edit through separate views. It would also mean that notes would have to be global to data sources, and couldn't be customized on a per-chart basis (whereas the short description field can).

What do you think about opening this as a separate issue and raising it with UCD? I'd be curious to hear if they'd rather have more flexibility as with the short description field, or if they'd rather save time with preset notes.

Reply:
Raising this with UCD sounds like the right route here to me too—maybe we can add it to our MVP demo agenda. 

Could you say more about #2 above when you open the issue? I don't quite understand what you mean. 

I don't think it makes sense to task the user with noting how the site is using ACS data, as they wouldn't necessarily know what's being pulled in or how it's being fit to the questions. For that reason, I think it makes sense to have ACS data notes as a global caption on a field wherever it's used, and treated as a separate attribute from the Short Description box, which I've been thinking of as more of a survey-specific caption.

Reply:
Opened up #160 to track this and added an agenda item to our MVP call to make sure we discuss it! By "exposed for list/detail/edit through separate views" I mostly meant that if we want to allow the user to create, edit, or delete these kinds of notes, we'll need corresponding views for it. Not a huge deal, of course, it'll just require a couple extra hours of work and some thinking about how to integrate it with the existing menu.

Ultimately I think you're right, UCD will almost certainly want these attributes and they'll want to be able to edit them. But mostly I just want to hear them vocalize that before putting the work into it.